docs/features/autoupdater.rst

```diff
@@ -11,14 +11,19 @@ during development), but it can be enabled by setting the variable GLUON_BRANCH
 to override the default branch set in the set in the site configuration.
 
 A manifest file for the updater can be generated with `make manifest`. A signing script (using
-ecdsautils) can by found in the `contrib` directory. When creating the manifest, ``GLUON_PRIORITY`` can
-be set on the command line, or it can be taken from the ``site.mk``.
+``ecdsautils``) can by found in the `contrib` directory. When creating the manifest, PRIORITY can
+be set on the command line, or it can be taken from the ``site.mk``, by the variable ``GLUON_PRIORITY``.
```

Comment:
This sounds as if the variable should be called `PRIORITY` when given on the command line. Just keep using `GLUON_PRIORITY` and don't talk about `PRIORITY` at all.